widgets/post-carousel/tpl/carousel-post-loop.php

```diff
@@ -4,15 +4,16 @@
  * @var string $default_thumbnail
  */
 while($posts->have_posts()) : $posts->the_post(); ?>
-	<div class="sow-carousel-item">
+	<div class="sow-carousel-item<?php if( is_rtl() ) echo ' rtl' ?>">
```

Comment:
Small fix here `if( is_rtl() )` changed to `if ( is_rtl() )`

Reply:
That's actually no longer required so I'm just going to remove it.

Reply:
Thanks!